json/from_json.mbt

```diff
@@ -122,6 +123,11 @@ pub impl[X : FromJson] FromJson for FixedArray[X] with from_json(json, path) {
 }
 
 pub impl[V : FromJson] FromJson for Map[String, V] with from_json(json, path) {
+  // Note in theory, we can make it more general like
+  // Map[K : FromString, V : FromJson]
+  // but currently we don't have FromString 
+  // Here ToJSON we have 
+  // Map[K : Show, V : ToJson]
```

Comment:
The comment `Map[K : Show, V : ToJson]` doesn't match the signature `Map[String, V : ToJson]`?

Reply:
it does, can you double check?